### ISSUE TITLE
Improvment for css background property

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#42]
```
В HTML есть удобная комбинация "Output Image Size" для 
вставки widht/height
текущего изображения.

Я часто вставляю изображения не через IMG, а 
с помощью DIV, указывая файл
через background:

.someimage{
background("i/img.png") no-repeat;
width: 100px;
height: 50px;
}

Возможно ли сделать такую же комбинацию 
для CSS, чтобы width и height
вставлялись автоматически?

Спасибо.


```

---

Original issue reported on code.google.com by `metaller...@gmail.com` on 23 Nov 2009 at 10:47

**Comment by GoogleCodeExporter** [comment]
```
Да, хорошая мысль. Запланирую на следующую 
версию
```

---

Original comment by `serge....@gmail.com` on 23 Nov 2009 at 10:53
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Сергей, когда же ждать новую версию? Очень 
востребованная и ожидаемая фича.
```

---

Original comment by `metaller...@gmail.com` on 10 Dec 2009 at 4:17

**Comment by GoogleCodeExporter** [comment]
```
Пока не знаю. Эта фича появится в 
какой-нибудь тестовой сборке, я напишу об 
этом.
```

---

Original comment by `serge....@gmail.com` on 10 Dec 2009 at 4:26

**Comment by GoogleCodeExporter** [comment]
```
А реализован ли данный функционал в 
недавно вышедшей версии 0.5.5/0.5.6 ?
```

---

Original comment by `metaller...@gmail.com` on 18 Jan 2010 at 9:30

**Comment by GoogleCodeExporter** [comment]
```
Нет ещё
```

---

Original comment by `serge....@gmail.com` on 18 Jan 2010 at 10:36

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `serge....@gmail.com` on 9 Feb 2010 at 10:54
- Added labels: **Milestone-0.6**

**Comment by GoogleCodeExporter** [comment]
```
Переношу на следующий майлстоун, так как 
надо сначала сделать нормальный CSS парсер
```

---

Original comment by `serge....@gmail.com` on 9 Feb 2010 at 8:56
- Added labels: **Milestone-0.7**
- Removed labels: **Milestone-0.6**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.7
```

---

Original comment by `serge....@gmail.com` on 1 Feb 2011 at 10:45
- Changed state: **Fixed**
